<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/icons.qrc">
    <normaloff>:/icons/wicon.png</normaloff>:/icons/wicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabText">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_text"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QToolButton" name="toolButton_textNew">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_textOpen">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_textSave">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_textSaveAs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabScreen">
       <attribute name="title">
        <string>Screen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea_image">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>721</width>
               <height>544</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_positionControls">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Screen</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_screenWidth">
                  <property name="text">
                   <string>Width</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_screenHeight">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Height</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinBox_screenWidth">
                  <property name="minimum">
                   <number>320</number>
                  </property>
                  <property name="maximum">
                   <number>1920</number>
                  </property>
                  <property name="singleStep">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>1024</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBox_screenHeight">
                  <property name="minimum">
                   <number>240</number>
                  </property>
                  <property name="maximum">
                   <number>1200</number>
                  </property>
                  <property name="value">
                   <number>768</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="pushButton_screenBackgroundColor">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_screenBackgroundColor">
                  <property name="text">
                   <string>Color</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Spacing</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinBox_spaceLeft">
                  <property name="maximum">
                   <number>1920</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBox_spaceRight">
                  <property name="maximum">
                   <number>1920</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_spaceBottom">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Bottom</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_spaceRight">
                  <property name="text">
                   <string>Right</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBox_spaceBottom">
                  <property name="maximum">
                   <number>1200</number>
                  </property>
                  <property name="value">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_spaceLeft">
                  <property name="text">
                   <string>Left</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Text</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_textFont">
                  <property name="text">
                   <string>Font</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_textPixelSize">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Pixel size</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QFontComboBox" name="fontComboBox_textFont"/>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBox_textPixelSize">
                  <property name="minimum">
                   <number>8</number>
                  </property>
                  <property name="maximum">
                   <number>1200</number>
                  </property>
                  <property name="value">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_screenTextColor">
                  <property name="text">
                   <string>Color</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="pushButton_screenTextColor">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Frame per second</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Repeat count</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>24</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>25</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>29.97</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>30</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>59.94</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>60</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Show guide lines (only in the preview)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_frameNumber">
            <property name="text">
             <string>Frame:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_frameNumber">
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>21</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_frameNumber">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_frameNumberOf">
            <property name="text">
             <string>of</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_frameCount">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRender">
       <attribute name="title">
        <string>Render</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Configure"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-about.png</normaloff>:/icons/action-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-about-qt.png</normaloff>:/icons/action-about-qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_Configure">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-configure.png</normaloff>:/icons/action-configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure...</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/file-save-as.png</normaloff>:/icons/file-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="action_textNew">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/file-new.png</normaloff>:/icons/file-new.png</iconset>
   </property>
   <property name="text">
    <string>New text...</string>
   </property>
   <property name="toolTip">
    <string>New text</string>
   </property>
  </action>
  <action name="action_textOpen">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/file-open.png</normaloff>:/icons/file-open.png</iconset>
   </property>
   <property name="text">
    <string>Open text...</string>
   </property>
  </action>
  <action name="action_textSave">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/file-save.png</normaloff>:/icons/file-save.png</iconset>
   </property>
   <property name="text">
    <string>Save text...</string>
   </property>
   <property name="toolTip">
    <string>Save text</string>
   </property>
  </action>
  <action name="action_textSaveAs">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/file-save-as.png</normaloff>:/icons/file-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save text as...</string>
   </property>
   <property name="toolTip">
    <string>Save text as</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>plainTextEdit_text</tabstop>
  <tabstop>toolButton_textNew</tabstop>
  <tabstop>toolButton_textOpen</tabstop>
  <tabstop>toolButton_textSave</tabstop>
  <tabstop>toolButton_textSaveAs</tabstop>
  <tabstop>scrollArea_image</tabstop>
  <tabstop>spinBox_screenWidth</tabstop>
  <tabstop>spinBox_screenHeight</tabstop>
  <tabstop>pushButton_screenBackgroundColor</tabstop>
  <tabstop>spinBox_spaceLeft</tabstop>
  <tabstop>spinBox_spaceRight</tabstop>
  <tabstop>spinBox_spaceBottom</tabstop>
  <tabstop>fontComboBox_textFont</tabstop>
  <tabstop>spinBox_textPixelSize</tabstop>
  <tabstop>pushButton_screenTextColor</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>horizontalSlider_frameNumber</tabstop>
  <tabstop>spinBox_frameNumber</tabstop>
 </tabstops>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
